INSTALL.md

```diff
@@ -22,6 +22,8 @@
 
 * Surelog Source code
   * git clone https://github.com/alainmarcel/Surelog.git
```

Comment:
You can also do `git clone --recursive https://github.com/alainmarcel/Surelog.git`, then you don't need the submodule init below.